cylc/flow/data_messages.proto

```diff
@@ -117,25 +140,13 @@ message PbJob {
     optional string finished_time = 8;
     optional string job_id = 9;
     optional string job_runner_name = 10;
-    optional string env_script = 11;
-    optional string err_script = 12;
-    optional string exit_script = 13;
-    optional float execution_time_limit = 14;
     optional string platform = 15;
```

Comment:
Left this in, so current API doesn't break.

Reply:
Makes sense to keep this as the job platform is chosen based on the runtime configuration (e.g. `platform` could be set to a platform-group) causing these two values to differ.